### DISCUSSION TITLE
PII data classification

### DISCUSSION THREAD

**Discussion by Shweta0703** [microsoft/presidio#649] [Q&A]
We need to look for PII data in database tables (column names and its data) using Presidio. We are planning to build using the below steps.
1. Connect with any database like postgreSql, mysql and get all the tables and it's column details.
2. Identify PII data based on column name and its data, based on a sample size of x rows (After concatenating the rows into single very big text)
3. Identify the PII response having maximum score

Please suggest whether these steps are correct and what API we should use? Should we use different ENTITY_TYPE to check the highest scoring model for each columns?

Looking for suggestions. Appreciate for suggestions of any other open source solution which will be good for this requirement. Thanks in advance !!

**Comment by omri374** [comment]
Hi @Shweta0703, there are some relevant discussions in this issue: #600
